extensions/mssql/src/mssql.d.ts

```diff
@@ -1379,6 +1379,23 @@ declare module 'mssql' {
 			containmentTypes: string[];
 			recoveryModels: string[];
 		}
+
+		export interface DatabasePropertiesInfo extends SqlObject {
```

Comment:
there is already a database dialog created by Cory, why do we have to have another one? currently we use one dialog for each object type and you can enable/disable the components based on whether it is creating new or editing existing objects.

Reply:
Thought of using the one created by Cory, but that is for create/delete database and will be having a huge no of updates on properties and thought it would be nice to maintain its code and properties separatley!

Reply:
that is for create database and it has a lot less options for now, but eventually they should reach parity.  I'd recommend to render based on the IsNewObject property. so that you don't have to change the way OM framework works.

Reply:
@ssreerama Can you work with @corivera and @erinstellato-ms to figure out how feasible that is? The SSMS dialogs are very different for Databases, but what Alan suggested would make integrating with the existing ADS stuff a lot easier. And having a single dialog could have other benefits as well.

Make sure you consider that there is likely going to be changes needed on the STS side to support separate dialogs if we decide to stick with that.

Reply:
Scratch what I said above about the dialogs being hugely different in SSMS - I thought their layout was completely different but it's just the same as the other types where we just hide the unnecessary fields from the create dialog. So what Alan said makes a lot of sense - we should definitely be using a single dialog for both cases.

Reply:
Yeah, I guess if I use the newObject property, I can differentiate between create database and properties dialog, and no need to have a string enum to differentiate them. made the PR draft and will work on reuse accordingly.